static/app/views/settings/account/accountIdentities.tsx

```diff
@@ -91,4 +194,54 @@ const IdentityPanelItem = styled(PanelItem)`
   justify-content: space-between;
 `;
 
+const InternalContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+const IdentityText = styled('div')<{isSingleLine?: boolean}>`
+  height: 36px;
+  display: flex;
+  flex-direction: column;
+  justify-content: ${p => (p.isSingleLine ? 'center' : 'space-between')};
+  margin-left: ${space(1.5)};
+`;
+const IdentityName = styled('div')`
+  font-weight: bold;
+`;
+const IdentityDateTime = styled(DateTime)`
+  font-size: ${p => p.theme.fontSizeRelativeSmall};
+  color: ${p => p.theme.gray300};
+`;
+
+const TagWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-grow: 1;
+  margin-right: ${space(1)};
+`;
+
+const Tag = styled(
```

Comment:
Can you just use a real `Tag` component for this https://storybook.sentry.dev/?path=/story/components-tags--basic